src/test/java/teammates/test/cases/webapi/GetInstructorPrivilegeActionTest.java

```diff
@@ -20,16 +31,329 @@ protected String getRequestMethod() {
         return GET;
     }
 
-    @Test
+    @Override
+    protected void prepareTestData() {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+        String section1 = dataBundle.students.get("student1InCourse1").getSection();
+        String session1 = dataBundle.feedbackSessions.get("session1InCourse1").getFeedbackSessionName();
+        InstructorPrivileges privileges = instructor1ofCourse1.privileges;
+        // update section privilege for testing purpose.
+
+        // course level privilege
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE, true);
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT, true);
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION, false);
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR, false);
+
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS, false);
+
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS, false);
+        privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS, false);
+        privileges.updatePrivilege(
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS, false);
+
+        // section level privilege
+        privileges.updatePrivilege(section1,
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS, true);
+        privileges.updatePrivilege(section1,
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS, true);
+
+        // session level privilege
+        privileges.updatePrivilege(section1, session1,
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS, true);
+        privileges.updatePrivilege(section1, session1,
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS, true);
+
+        instructor1ofCourse1.privileges = privileges;
+
+        instructor1ofCourse1.role = Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM;
+        dataBundle.instructors.put("instructor1OfCourse1", instructor1ofCourse1);
+        removeAndRestoreDataBundle(dataBundle);
+    }
+
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        // see individual tests.
+    }
+
+    @Test
+    protected void testExecute_fetchPrivilegeOfNonExistInstructor() {
```

Comment:
You can add a description for expected behaviour at the end of the method. `testXX_xxxx_shouldReturnNotFoundError`. Same for other places when applicable.